Source/s3stream/AmazonS3Client.cpp

```diff
@@ -322,3 +322,30 @@ Framework::Http::RequestResult CAmazonS3Client::ExecuteRequest(const Request& re
 #endif
 	return response;
 }
+
+ListObjectsResult CAmazonS3Client::GetListObjects(std::string accessKeyId, std::string secretAccessKey, std::string bucketName)
```

Comment:
This should be moved in `ui_shared`. I'd like `AmazonS3Client` to stay a thin layer over S3's REST API (https://docs.aws.amazon.com/AmazonS3/latest/API/API_Operations_Amazon_Simple_Storage_Service.html). We could create a `CAmazonS3Utils` class in the ui_shared module to host that function.